easy-ui-react/src/DropdownButton/DropdownButton.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactNode, useRef } from "react";
+import ExpandMoreIcon from "@easypost/easy-ui-icons/ExpandMore";
+import { useButton } from "react-aria";
+import { Icon } from "../Icon";
+import { ButtonColor } from "../types";
+import { classNames, variationName } from "../utilities/css";
+import { LogWarningIfInvalidColorVariantCombination } from "../utilities/button";
+
+import styles from "./DropdownButton.module.scss";
+
+export type DropdownButtonVariant = "filled" | "outlined";
+
+export type DropdownButtonProps = {
+  /** Button color */
+  color?: ButtonColor;
+  /** Button variant */
+  variant?: DropdownButtonVariant;
+  /** Disables button */
+  isDisabled?: boolean;
+  /** Content inside button  */
+  children?: ReactNode;
+};
+
+export function DropdownButton(props: DropdownButtonProps) {
+  const {
+    color = "primary",
+    variant = "filled",
+    isDisabled = false,
+    children = "Button",
+  } = props;
+
+  const ref = useRef(null);
+  const { buttonProps } = useButton(props, ref);
+
+  LogWarningIfInvalidColorVariantCombination(color, variant);
+
+  return (
+    <button
+      disabled={isDisabled}
+      ref={ref}
+      className={classNames(
+        styles.DropdownButton,
+        styles[variationName("color", color)],
+        styles[variationName("variant", variant)],
+      )}
+      {...buttonProps}
+    >
+      <span>{children}</span>
+      <div className={classNames(styles.pipeSeparator)}></div>
```

Comment:
off the top of your head, do you know if there are any accessibility gotchas here with the pipe and empty div?

Reply:
i don't believe there are any accessibility gotchas with styling an empty div. if there was text inside it would need to be visually hidden, but it's empty here so should be fine.

nit: i'd suggest making it a `span` instead of a `div` to denote that it's `inline` instead of `block`